misc/lldb_rb/commands/print_flags_command.py

```diff
@@ -0,0 +1,30 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class PrintFlagsCommand(RbBaseCommand):
+    program = "print_flags"
+
+    help_string = "Print out the individial flags of an RVALUE object in human readable format"
+
+    # call is where our command logic will be implemented
+    def call(self, debugger, command, exe_ctx, result):
+        g = super().lldb_init(debugger)
```

Comment:
Why wasn't `lldb_init` called here? It should have been called by `__call__` right?